### DISCUSSION TITLE
1-1. 세상이 정의하는 개발자

### DISCUSSION THREAD

**Discussion by imakerjun** [ssac-dev/yozm#3] [1장 이런 개발자가 되고 싶어]
# 1-1. 세상이 정의하는 개발자
## 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기
일상에서 자주 사용하는 소프트웨어를 다섯 가지 적어보세요. 예를 들어, 쿠팡이나 컬리와 같은 온라인 커머스 앱, 카카오톡과 같은 메신저, 유튜브와 틱톡 같은 영상 콘텐츠 앱, 배달 음식 앱, 슬랙과 같은 협업 도구 등이 있습니다. 이러한 소프트웨어는 우리 생활을 크게 바꾸었고, 개발자의 중요성을 보여줍니다.

## 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기
불확실성이 높은 미래 사회에서는 변화하는 상황에 발맞추어 좋은 서비스를 만들어낼 수 있는 개발자의 중요성이 점점 더 높아지고 있습니다. 이와 같이 개발자의 수요가 점점 높아지는 이유를 적어보세요.

## 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기
변하지 않는 한가지가 있다면, 무엇이던 변화한다는 것입니다. 이 변화 중 AI 기술과 같이 우리 삶의 형태, 루틴까지 변화시킬 가능성을 가진 큰 변화들도 있습니다. 챗 GPT와 같이 요새 큰 화두가 되고 있거나, 중요한 변화라 생각하는 변화를 한가지 이상 적어보세요.

**Comment by imakerjun** [comment]
# 1. 메이커준에게 영향을 끼친 소프트웨어들 

생각해봤을때 저에게 영향을 끼친 것들 중 임팩트가 큰 것 5가지를 골라보자면, 여행에 관련된 것들이에요.
제 첫 해외 배낭 여행은 인도여행이었는데요. 그때 당시에는 스마트폰도 없었던터라 가이드북과 종이 지도를 보면서 숙소와 여행지를 찾아다녔어요.
그런데 지금은 소프트웨어 덕분에 너무나 간편하게 여행을 할 수 있게 되었고, 더 자주 갈 수 있게 된것 같아요.

- 스카이스캐너
- 그랩, 우버
- 구글맵
- 에어비엔비
- 팟캐스트

**Comment by kaori-killer** [comment]
## **1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기**

저는 집순이인데요, 제가 꼽은 5가지 소프트웨어는 저를 더 집순이로 만들었습니다 🏠  모든 게 집에서 가능한 세상이 만들어지고 있는 것 같아요. 저는 이게 편하면서도 너무 집에 있게 돼서 싫습니다. 😅 소프트웨어는 세상을 쉽게 연결을 해주면서 또 쉽게 단절시키는 것 같아요.

- 티빙
- 카카오톡
- 유튜브
- 인스타그램
- 배달의민족

## **2. 개발자 수요가 높은 이유 한 가지 이상 적어보기**

모든 기업은 고객의 피드백에 빠르게 대응하는 것을 원합니다. 그래서 IT가 도메인이 아닌 기업도 IT 역량을 보유하는 것이 중요한 요소가 되었어요. 다양한 분야에서 개발자를 필요로 하기 때문에 개발자의 수요가 올라가는 것 같습니다.

## **3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기**

소프트웨어의 발전이 삶의 양극화를 더욱 극명하게 만들고 있다는 점입니다. 소프트웨어의 발전으로 인해 저는 굉장히 편리함을 느끼고 있습니다. 하지만 발전을 따라가지 못하는 사람은 배달 서비스, 택시 호출, 전자 은행 업무 등 일상의 많은 부분에서 도태되고 있다는 것을 느낍니다. (하지만 소프트웨어가 더욱 발전하면 많은 사람을 포용할 수 있을 듯 싶어요.)

**Comment by shackstack** [comment]
## 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기

저는 이사를 하면서 라이프 스타일이 많이 바뀌었어요. 전처럼 대형마트가 근처에 없어 쿠팡을 애용하게 되었고 주방이 작아지면서 배달 주문량이 급격히 늘었습니다. 이런 소프트웨어가 없었다면 오우... 생각만해도 끔찍하네요 🙄... 

배달의민족
쿠팡
카카오톡
유튜브
넷플릭스

## 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기

많은 산업에서 지금까지도 꾸준히 디지털 전환이 발생하고 있습니다. 이러한 디지털 전환이 새로운 비즈니스 모델을 만들어주기도하고 마진을 남겨주기도하고 업무 생산성을 높여주기도하기 때문에 개발자 수요가 높은 것 같습니다.

## 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기

시간 대비 성능을 뜻하는 '시성비'가 요즘 트렌드인것 같아요. 식당 웨이팅 시간을 절약해주는 서비스, 숏폼, B마트 장보기, 한집 배달이 인기인걸 보면요. 극한의 효율을 추구하는 저로서는 이러한 트렌드를 반영한 서비스들이 좋지만요, 한편으로 낭만이 없어지는 것 같아 아쉽네요~

**Comment by baobabnamu** [comment]
## 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기
군대의 체력단련실에서 사용하는 애플리케이션, 친구들과 연락할 때 사용하는 애플리케이션, 연등 시간에 책 읽을 때 사용하는 애플리케이션 정도가 있네요. 이 고마운 친구들 덕분에 군대에서도 편리하게 자기개발 할 수 있어서 행복합니다 :)
- Gymwork
- Youtube Music
- Kakaotalk
- Instagram
- E-Book (알라딘, 밀리의 서재)

**Comment by baobabnamu** [comment]
## 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기
4차산업혁명이 도래함에 따라 많은 산업들이 이미 융합이 진행되고 있습니다. 그 융합의 중점이 되는 기술이 바로 IT 기술입니다. 누구나 훌륭한 커리어 패스를 가지고 싶은만큼 현재 주목받고 있는 기술에 다양한 직군 사람들이 모일 수 밖에 없는 것 같습니다. 더군다나 자유로운 업무 환경을 보장해 주는 회사와 높은 보수를 제공하는 회사들도 늘고 있어서 더욱 경쟁이 심화되고 있는 것 같습니다. 과거에 없던 업무 방식이라서 더욱 사람들이 매력을 느끼고 도전하고 있는 거 아닐까 싶네요. :)

**Comment by baobabnamu** [comment]
## 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기
최근에 들어서 은행에 갈 일이 있었는데 대부분의 은행 지점들이 폐쇄되고, 디지털 출장소로 대체되는 것을 볼 수 있었습니다. 급속화된 디지털 흐름 아래, 수익성 악화를 이유로 대부분의 시중 은행이 점포 통폐합을 진행한 것이 이유였습니다. 앞으로도 간단한 사무직에 대한 인력 감축 변화는 계속될 것 같습니다. 급변하는 시대에 맞게 문제해결 능력을 길러서 사회에 걸맞은 인재가 되어야겠다는 생각이 듭니다. :)

**Comment by Zhyun18** [comment]
# 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기

- 유튜브
- 쿠팡
- 배달의민족
- 토스
- 당근

정말 생각나는대로 적어보았는데  크게 카테고리화해보면, 
영상 (유튜브)  / 쇼핑 (쿠팡/당근)  / 식사 (배민)  / 뱅킹(토스) 이 소프트웨어를 사용하면서 더욱 간편해지고, 제 소비 폭도 덕분에 늘었습니다. 
(이건 좋지 않은  것 같습니다) 

# 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기 

우선, 이용자의 증가 때문이 아닐까 생각합니다. 특히, 코로나 이후 급격하게 대면 업무가 비대면 업무로 바뀌면서, 
많은 사람들이 오프라인이 아닌 온라인에서 본인들의 업무를 보기 시작해서가 아닐까 싶네요. 

또한, 이제는 IT 기업 뿐만 아니라 다른 산업군에서도 DX 전환을 목표로 변하고 있기 때문에 디지털 역량을 갖춘 인력의 수요가 늘은 것 같습니다.
반복적인 업무의 자동화를 넘어선, chatGPT 와 같이 생성형 AI 를  현업에 활용하는 시대이기 때문에 개발자의 수요는 점점 증가하고 있다 생각합니다. 

# 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기 

개인의 취향을 마음껏 드러내고, 취향에 따라 개별적인 소비가 일어나는  것 같습니다. 소비자의 선택의 폭이 커진만큼, 본인을 자유롭게 표현하는데에 
거리낌 없는 사회 분위기에서 기업들은 사용자의 취향에 맞는 콘텐츠/ 사용자의 구매패턴을 예측하여 추천 및 프로모션을 제공합니다. 
그만큼 개인의 소비 폭이 이전보다 더욱 증가한 느낌입니다. 똑똑해진 기업이 소비자를 잘 놓치지 않는 것 같고요

**Comment by heeji289** [comment]
# 1-1. 세상이 정의하는 개발자
## 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기
- 크롬 브라우저
- 카카오톡
- 인스타그램
- 유튜브
- 쿠팡이츠

카카오톡이랑 유튜브 없이는 못 살 것 같네요

## 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기
아무래도 무언가를 직접적으로 만들어내는 역할을 하기 때문에 수요가 높은 것 같아요. 누군가가 기획을 하고 디자인을 하면 실제로 생산해내는 주체라서? 
요즘 기술들도 이전보다 많이 좋아졌다고 하고 개발자를 잘 채용하면 할 수 있는게 많아서 그런게 아닐까 합니다.

## 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기
LLM을 활용한 서비스가 말도 안되게 많이 나오고 있습니다. 요즘에 본 독특한 서비스는 LLM을 활용한 캐릭터 챗봇 앱, 소설 자동완성 앱 등 등이 있네요.
깊이 이해하지 못해도 어느정도 활용은 할 줄 알아야 사는 게 편할 것 같습니다.

**Comment by K-julius** [comment]
> ## 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기
> 유튜브
> 카카오톡
> 네이버맵
> KG 이니시스(전자결제 서비스)
> 쿠팡 
>
> ## 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기
> 온라인 서비스가 너무나 익숙하고 당연해진 시대가 되었다고 생각합니다. 사람들은 종종 카카오톡과 인스타그램이 나오기 전 개개인의 안부를 어떻게 묻고 어떻게 연락해왔는지 잊곤 하며, 길을 찾을 때 맵 서비스를 활용하는 것도, 휴대폰으로 전자결제를 하는 것도 당연해졌습니다. 하지만 온라인 서비스는 당연하지만, 각각의 온라인 서비스가 무조건적으로 필요하지는 않습니다. 사람들은 여러 소프트웨어를 사용하고 그만큼 많은 소프트웨어를 잠깐 이용했다가 지우기도 하고 잊으며, 새로운 것을 사용하기도 합니다. 
 그럼에도 불구하고 오프라인과 비교도 안될 정도의 유동 인구가 온라인에 존재합니다. 아마존과 구글, 메타처럼 21세기 성공한 기업에 소프트웨어 관련 기업들도 다량 분포돼 있습니다. 이에 따라 21세기에 돈을 벌고 싶다면 온라인 서비스는 필수적이어 보이고, 어렵게 모이고 쉽게 떠나가는 소비자들의 선택에 민감하게 반응하며 즉각적으로 서비스를 만들 수 있는 사람들이 필요해진 것으로 보입니다.
> 
> ## 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기
> 코로나로 세상이 떠들썩 했을 때, 비대면 서비스가 더욱 확산된 것이 꽤 중요한 변화로 보입니다. 이전까지 존재했던 기술의 변화를 익숙함에 의지하느라 사용하지 않고 미루어 두었었습니다. 하지만 비대면 서비스가 필수적인 사회가 되자 온라인 서비스의 편리함을 느끼게 되었고, 이후에도 온라인 서비스를 계속적으로 사용하게 만들었다고 생각합니다. 비대면 서비스가 필수적인 상황이 아니었더라면, 이 정도로 온라인 서비스에 의지하는 시대가 조금은 늦춰지지 않았을까 생각해봅니다.

**Comment by Bigstargithub** [comment]
## 내 일상생활에 많은 영향을 끼치는 소프트웨어
### 링크드인
: 링크드인은 전 세계 사람들의 커리어 관련 생각이나 근황을 볼 수 있는 곳입니다. 저는 이 곳에서 저의 커리어 관련 소식이나 회사 생활하면서 겪었던 느낌들을 정리하고 있습니다.

### 카카오뱅크
: 카카오뱅크는 카카오톡과의 연동이 좋아서 경조사 관련 비용을 보내고 생활비 용도로 사용하고 있습니다.

### 리그오브레전드(LOL)
: 리그오브레전드는 라이엇 게임사가 만든 전 세계적으로 유명한 게임입니다. 머리가 아프고 쉴 때 한 판씩 하고 있습니다.

### 티맵
: 제가 출퇴근할 때 운전해서 역까지 가는데, 그럴 때 티맵을 켜서 과속 단속 지역을 알고 티맵은 지금 현재 교통상황을 반영해서 제일 빠른 길로 안내하기 때문에 저에게 영향을 끼치고 있습니다.

### 유튜브
: 예전에는 정보를 블로그의 글로 얻었지만, 요즘은 유튜브 매체를 통해서 영상으로 얻고 있습니다. 저도 필요한 정보가 있으면 유튜브에 검색해서 정보를 얻고 있습니다.

## 개발자 수요가 높은 이유
: 개발자 수요가 높은 이유는 IT 기술을 통해서 회사의 가치를 더 많은 사람에게 전달할 수 있기 때문에 지금 개발자의 수요가 높다고 생각합니다.

## 내가 중요하다고 생각하는 세상의 변화 한 가지 이상 적어보기
: 스마트폰의 등장으로 손 안에서 많은 일들을 할 수 있다는 변화입니다. 그렇기에 개발자라는 직종이 급부상한 계기가 아닐까 싶습니다.

**Comment by gogo1414** [comment]
## 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기
- 카카오맵, 구글맵
- 카카오톡
- 유튜브, 인스타

제가 심각한 길치인지라... 처음 보는 장소라면 바로 길을 잃을 가능성이 높아 이동 시에는 무조건 지도를 참고하고 다녀서 카카오맵과 구글맵과 같은 소프트웨어에 많이 의지하고 있습니다.

## 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기
저는 기존에 제가 사용한 방식보다 더 편한 방식이 존재하면 그 방식으로 쉽게 이동하곤 합니다. 이러한 방식을 제공하는 소프트웨어를 개발하는 사람이 개발자이기에 개발자 수요가 높아지고 있다고 생각합니다.

## 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기
스마트폰이 없던 전의 삶과 있고난 후의 삶의 방식이 확연히 달라진게 느껴져서 스마트폰의 존재가 세상의 중요한 변화 중 하나라고 생각합니다.

**Comment by dongjooyun** [comment]
# 1-1. 세상이 정의하는 개발자
## 1. 내 일상생활에 많은 영향을 끼치는 소프트웨어 다섯 가지 적어보기
### 1. 토스
- 유일하게 살아남은 짠테크 앱
- 디자인이 예뻐서 쓰게 된다
### 2. 네이버지도
- 길치에게 필수 앱
- 경로 저장 기능이 유용
### 3. 삼성헬스, 플랜핏, NRC, 런데이
- 운동 관련 앱들
- 오운완 SNS 공유 기능 필수
### 4. 투두메이트
- 게으른 P를 일으켜 세운 유일한 투두 앱
### 5. 노션
- 에버노트 → MS원노트 쓰던 나에게 혁신
- 모든 기록 저장소

## 2. 개발자 수요가 높은 이유 한 가지 이상 적어보기
1. 온 세상이 SW니까 DT할 사람이 필요하니까
2. 누구나 SW를 만들 수 있는 세상이 됐지만 ‘비용 최소화 효율 극대화’하며 프로그래밍할 사람이 필요하니까

## 3. 내가 느끼는 세상의 중요한 변화 한 가지 이상 적어보기
1. 출산률 감소 & 고령화
2. 얼어붙은 취업시장
3. 미국 정세
4. AI와 로봇의 발전
5. SNS
